modules/ROOT/pages/design-publish.adoc

```diff
@@ -0,0 +1,47 @@
+
+.The page that Exchange creates for a published project includes a left pane for viewing the specification's documentation, a middle pane for viewing a description of the specification, and a right pane for viewing metadata about the specification.
+image:apid-example-page-in-exchange.png[An example page for a specification published in Exchange]
+
+<1> The left pane, which you can navigate to view the documentation. The navigation is generated from the documentation in the specification.
+<2> The middle pane, which describes the specification. You write the description in Exchange after publishing the specification.
+<3> The right pane, which displays metadata about the specification.
+
+== Procedure
+
+. Open the API-specification project that you want to publish, if it is not already open.
+. In the top-right corner of API Designer, click *Publish* and then click *Publish to Exchange*.
+
+image:apid-publish-dialog-api-specs.png[]
+
+<1> *Asset version*: The version number to be displayed in the *Version* section of the page that Anypoint Exchange creates for the project.
+<2> *API version*: The version number of the API specification. For a description of this field, see the next section.
+<3> *Business group ID*: The ID of the business unit in Anypoint Platform to which the project belongs.
+<4> *Asset name*: The name that Anypoint Exchange uses to identify the published project.
+<5> *Asset ID*: The ID that Anypoint Exchange uses to identify the published project.
+<6> *Add to API Manager*: Select this option if you want to publish the API specification to an API portal. When you click *Publish to Exchange*, API Designer also publishes the specification to API Manager, where the specification is listed in the *API Administration* section.
+
+== The *API version* field
+
+The appearance of the *API version* field depends upon several factors:
+
+* Anypoint Exchange requires that each API specification published to it have a version number.
+* API Manager requires that API specifications provide a version number.
+* In specifications written in OAS, the `version` field is required.
+* In specifications written in RAML, the `version` node is optional.
+
+The following flowchart shows how API Designer decides whether to disable or enable the *API version* field and whether to populate it with a number.
+
+.The appearance of the *API version* field depends on the outcome of the checks that API Designer makes when you want to publish an API specification.
+image:apid-flowchart-api-version.png[Flowchart for the checks that API Designer makes]
+
+<1> API Designer checks whether the version number for the API specification is provided within the specification. If the version number is provided, then the *API version* field is populated with the number and disabled, so that you cannot change the number. If the specification does not provide the version number, then API Designer moves to the next check.
```

Comment:
GLOBAL for this list:
```suggestion
<1> API Designer checks whether the version number for the API specification is provided within the specification. 
If the version number is provided, then the *API version* field is populated with the number and disabled, so that you cannot change the number. If the specification does not provide the version number, then API Designer moves to the next check.
```

Reply:
@kbostrom , can't do this with Asciidoc callouts.